api/setting.go

```diff
@@ -50,48 +38,48 @@ var (
 		MigrationSurchargeMultiplier:  10,                                               // 10x
 	}
 
-	// DefaultPricePinSettings define the default price pin settings the bus is
-	// configured with on startup. These values can be adjusted using the
-	// settings API.
-	DefaultPricePinSettings = PricePinSettings{
+	// DefaultPinnedSettings define the default pin settings the bus is
+	// configured with on startup.
+	DefaultPinnedSettings = PinnedSettings{
```

Comment:
With the introduction of an explorer section and making the forex endpoint a static configuration, it probably makes sense to merge the other PR first right?

Then we can get rid of the `Enabled` and `ForexEndpointURL` in the pinning settings.

Reply:
We can only get rid of `Enabled` if we implicitly use `currency` as an enabled yes/no (which I did not like). Is that what you are proposing? Pinning can still be enabled/disabled regardless of the status of the explorer right. Not all states obviously but I can enable/disable it as long as the explorer is enabled.

Reply:
Get what you mean now!